### PR TITLE
IJPS 50

### PR DESCRIPTION
**Pull request by Josh-HMCTS**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/IJPS-50

### Change description ###

Add delete functionality to view sitting records UI 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: src/app/_models/addSittingRecords.model.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export class SittingRecordsPostObj {
+    hmctsServiceCode!: string;
+    sittingDate!: Date;
+    epimsId!: string;
```
Comment (markdathornehmcts) [inline src/app/_models/addSittingRecords.model.ts:4]: shouldn't this be epimmsId?

File: src/app/_services/delete-sitting-records-http-service.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable, map } from 'rxjs';
```
Comment (markdathornehmcts) [inline src/app/_services/delete-sitting-records-http-service.ts:3]: Possibly resolve Github warning

File: src/app/_services/delete-sitting-records-http-service.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+@Injectable({
+  providedIn: 'root'
+})
+export class DeleteSittingRecordHttp {
+
+  private deleteRecordsURL = '';
+
+  constructor(private http: HttpClient) { }
+
+  deleteRecord(recordID: string): Observable<any> {
```
Comment (markdathornehmcts) [inline src/app/_services/delete-sitting-records-http-service.ts:14]: Possible resolve Github warning

File: src/app/_services/sitting-records-service/sitting-records.service.ts [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    postNewSittingRecord(newSittingRecords: SittingRecordsPostBody) {
+        const headers = {
+            'Content-Type': 'application/json',
+        };
+
+        return this.http.post('/sittingrecord/add', { sittingRecords: newSittingRecords }, { headers: headers});
+    }
+
+
+    createNewSRPostObj(joh: any, tribunalService: any, dateSelected: any, venue: any, period:any) {
```
Comment (markdathornehmcts) [inline src/app/_services/sitting-records-service/sitting-records.service.ts:27]: Possible resolve Github warning

File: src/app/_services/sitting-records-service/sitting-records.service.ts [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+        return {
+            hmctsServiceCode: tribunalService.hmctsServiceCode,
+            sittingDate: this.dateSvc.formatDateForPost(dateSelected),
+            epimsId: venue.epimms_id,
```
Comment (markdathornehmcts) [inline src/app/_services/sitting-records-service/sitting-records.service.ts:31]: Change to epimmsId

**Review by markdathornehmcts — CHANGES_REQUESTED** [review]
Approved but with suggested changes

File: src/app/_models/addSittingRecords.model.ts (hunk 1 above)
Comment (Josh-HMCTS) [reply src/app/_models/addSittingRecords.model.ts:4]: will be resolved as part of merge with IJPS-43

File: src/app/_services/delete-sitting-records-http-service.ts (hunk 2 above)
Comment (Josh-HMCTS) [reply src/app/_services/delete-sitting-records-http-service.ts:3]: remove unused imports

File: src/app/_services/sitting-records-service/sitting-records.service.ts (hunk 4 above)
Comment (Josh-HMCTS) [reply src/app/_services/sitting-records-service/sitting-records.service.ts:27]: will be resolved with IJPS-43 merge

File: src/app/_services/delete-sitting-records-http-service.ts (hunk 3 above)
Comment (Josh-HMCTS) [reply src/app/_services/delete-sitting-records-http-service.ts:14]: Updated any

**Review by RebeccaBaker — APPROVED** [review]

**Pull request merged**